ChangeLog

```diff
@@ -107,6 +107,10 @@ well as interpreting \b and \v as characters.
 24. Fixed a bug in JIT affecting greedy bounded repeats. The upper limit of
 repeats inside a repeated bracket might be incorrectly checked.
 
+25. Fixed a bug in JIT affecting caseful matching of backreferences. When
+utf is disabled, and dupnames is enabled, casesless matching is used instead
```

Comment:
When you say "caseless matching **is** used" do you mean, "caseless matching was used, and the behaviour has now been fixed to be caseful"?